config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_secretmanagersecrets.secretmanager.cnrm.cloud.google.com.yaml

```diff
@@ -243,48 +299,49 @@ spec:
                         - external
                       properties:
                         external:
-                          description: 'Allowed value: string of the format `projects/{{project}}/topics/{{value}}`,
-                            where {{value}} is the `name` field of a `PubSubTopic`
-                            resource.'
+                          description: If provided must be in the format `projects/[project_id]/topics/[topic_id]`.
                           type: string
                         name:
-                          description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+                          description: The `metadata.name` field of a `PubSubTopic`
+                            resource.
                           type: string
                         namespace:
-                          description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
+                          description: The `metadata.namespace` field of a `PubSubTopic`
+                            resource.
                           type: string
                       type: object
                   required:
                   - topicRef
                   type: object
                 type: array
               ttl:
-                description: |-
-                  Immutable. The TTL for the Secret.
-                  A duration in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s".
+                description: Input only. The TTL for the [Secret][google.cloud.secretmanager.v1.Secret].
                 type: string
               versionAliases:
                 additionalProperties:
-                  type: string
+                  format: int64
```

Comment:
So I think we've changed from `versionAliases map[string]string` to `versionAliases map[string]int` .  Can we add test coverage to check that we aren't breaking existing resources, in particular

````
versionAliases:
  foo: 1
```

and

````
versionAliases:
  foo: "1"
```

Reply:
Sure!